TinCan/InteractionComponent.cs

```diff
@@ -0,0 +1,59 @@
+/*
+ *  Created by Paul Carpenter 2015
+ *  
+ *  Includes support for Activity Interaction Components
+ *  
+ */
```

Comment:
See above comment about attributions. This file still needs the Apache license header to match the others.